department-of-veterans-affairs/caseflow-efolder
# Kev ma/appeals 45277 v2

Resolves https://jira.devops.va.gov/browse/APPEALS-45277

### Description
Migrate VBMS::Request.GetDocumentContent SOAP request to Claim Evidence Api GetDocumentContent endpoint.
Added FeatureToggle to use VeteranFileFetcher (from ruby_claim_evidence_api gem), which will return a PDF Byte String of the latest PDF file for a given document series id

### Acceptance Criteria
- [ x] Code compiles correctly

### Testing Plan
1. Delete temp.pdf file is it exist in /tmp folder
2. Run Record.fetch
3. PDF file should have been written into the folder

### Integrations: Adding endpoints for external APIs
* [ x] Check that Caseflow's external API code for the endpoint matches the code in the relevant integration repo
  * [x ] Request: Service name, method name, input field names
  * [x ] Response: Check expected data structure
* [ x] Update Fakes
* [ x] Integrations impacting functionality are tested in Caseflow UAT
